src/primary_backup/main.rs

```diff
@@ -72,16 +72,19 @@ async fn main() {
         Node::backup(
             &db_name(&cli, &format!("replic-{}", cli.network_port)[..]),
             network_address,
+            Vec::new()
```

Comment:
If running the binary from the command line, there is no way to run multiple nodes and pass the peers list, right? Unless I'm missing something, I think that should be added to the CLI arguments (the lock-commit implementation has something similar)

Reply:
For this pr to be finished I have to fix main